test/e2e/e2e_test.go

```diff
@@ -20,7 +20,7 @@ import (
 const (
 	serviceCAOperatorNamespace   = "openshift-service-ca-operator"
 	serviceCAControllerNamespace = "openshift-service-cert-signer"
-	serviceCAOperatorPodPrefix   = "service-ca-operator"
+	serviceCAOperatorPodPrefix   = "openshift-service-ca-operator"
```

Comment:
we should import these from the `operatorclient` package: https://github.com/openshift/service-ca-operator/blob/a92546fbaa0c89374c4b4981f57e5b44aae88590/pkg/operator/operatorclient/interfaces.go#L6